contributor/01-introduction-article.asciidoc

```diff
@@ -12,60 +12,52 @@ If you're not getting what you need, you can make the change you need directly i
 The Contributor role describes a person that makes contributions to the repos of an InnerSource community project.
 This person may or may not be part or see themselves as part of the community.
 However, for quite a few people there is a sort of journey that contributors can make from just knowing about the community to using the community's product to interacting with members of the community and finally starting to contribute.
-Finally, some of them may become [Trusted Committers][TC Introduction].
+Finally, some of them may become https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/trusted-committer/01-introduction.md[Trusted Committers].
```

Comment:
Should link to the `asciidoc` version of the files?

Reply:
892e8c3  should fix the markdown based links and move them over to ASCIIdoc.

Reply:
Caught and fixed a few more in 0511483.